OriginateTheme/Scripts/ot_generator.py

```diff
@@ -151,6 +151,33 @@ def createColorGetter(color, name, keyPath):
 }"""
     return string.Template(getter).substitute({ 'name' : name, 'colorKeyPathKey' : keyPath, 'colorCode' : color.code})
 
+def createBoolGetter(color, name, keyPath):
+    """
+        Creates a new getter to access a OriginateTheme UIColor property.
+        
+        Parameters
+        -----------
+        color: Object
+        Object of type 'Color'.
+        name: String
+        The name of the property to create a getter for.
+        keyPath: String
+        The name of the key path to associated to the property.
+        """
+    getter = """
+        - (UIColor *)$name
+        {
+        if (!_$name) {
```

Comment:
make sure to indent the generated code properly (following the conventions that are already being used)